src/marqo/core/index_management/vespa_application_package.py

```diff
@@ -530,7 +530,7 @@ def deploy_application(self) -> None:
 
 
 class VespaApplicationPackage:
-    """
+    r"""
```

Comment:
nit: typo

Reply:
`r` means raw string, this is to fix a warning about `\` character needs to be escaped after python 3.9 upgrade